### PR TITLE
Load from filename not stream

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: hocr-merge-dc [hunk 1]
```diff
@@ -34,7 +34,7 @@ if len(sys.argv)<2:
     sys.exit(1)
 
 dc_doc = etree.fromstring(open(sys.argv[1]).read())
```
Comment (zuphilip) [inline hocr-merge-dc]: What about this statement? Can you replace similarly?

Comment (kba) [inline hocr-merge-dc]: Yes. And possibly also make sure we [parse as XHTML](https://github.com/tmbdev/hocr-tools/commit/8cf1682e61b034fb8406016a7ac77a07bd4b9a17#diff-305fdec86283ac99b306636017404b40R28).

**Comment by zuphilip** [comment]
I suggest also to replace these two occurences: https://github.com/tmbdev/hocr-tools/blob/master/hocr-eval#L144-L145

File: hocr-merge-dc (hunk 1 above)
Comment (zuphilip) [inline hocr-merge-dc]: Done

**Comment by zuphilip** [comment]
Thank you @stweil !

**Pull request merged**